dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPVDiagramPQ.mo

```diff
@@ -18,15 +18,24 @@ model GeneratorPVDiagramPQ "Model for generator PV based on SignalN for the freq
   extends BaseClasses.BaseQStator(QStatorPu(start = QGen0Pu * SystemBase.SnRef / QNomAlt));
 
 equation
-  when QGenPu <= QMinPu and UPu >= URefPu then
+  when QGenPu + QDeadBand  <= QMinPu and UPu - UDeadBand >= URefPu then
```

Comment:
```suggestion
  when QGenPu + QDeadBand  <= QMinPu and UPu - UDeadBand > URefPu then
```
We could remove the equality here and at line 25 since it is taken into account in the following conditions at line 30 and 35

Reply:
Done